tests/providers/amazon/aws/transfers/test_gcs_to_s3.py

```diff
@@ -128,7 +128,65 @@ def test_execute_without_replace(self, mock_hook):
             assert [] == uploaded_files
             assert sorted(MOCK_FILES) == sorted(hook.list_keys("bucket", delimiter="/"))
 
-    # Test3: There are no files in destination bucket
+    # Test3: All the files (within some folders) are already in origin and destination without replace
```

Comment:
Let’s remove the TestX prefix, they provide no value at all and very messy to maintain as displayed in this PR.

Reply:
True. Removed!